graphql/resolvers.go

```
@@ -831,6 +831,79 @@ func (r *mutationResolver) RestartTask(ctx context.Context, taskID string) (*res
 	return apiTask, err
 }
 
+func (r *mutationResolver) SaveSubscription(ctx context.Context, subscription restModel.APISubscription) (bool, error) {
+	usr := route.MustHaveUser(ctx)
+	username := usr.Username()
+
+	var id string
+	var idType string
+	for _, s := range subscription.Selectors {
+		if s.Type == nil {
```

This should probably return an error rather than continue